### PR TITLE
fix: convert inline styles to objects and fix mdx formatting

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #

The Platform is using v2 of mdx which requires styles to be formatted as objects. 

#### Changelog

- Converted all inline styles to objects
- Fixed formatting in code mdx files to prevent [`Unexpected end of file in expression, expected a corresponding closing brace for {” error?`
](https://mdxjs.com/docs/troubleshooting-mdx/#unexpected-end-of-file-in-expression-expected-a-corresponding-closing-brace-for-)

### CODE REVIEW

File: src/pages/components/UI-shell-header/code.mdx [hunk 1]
```diff
@@ -69,7 +69,13 @@ the Storybooks for each framework below.
       src="https://codesandbox.io/embed/v10-starter-sandbox-kf3h4?fontsize=14&hidenavigation=1&view=preview"
       title="header-base"
       allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media"
-      style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+      sstyle={{
```
Comment (joshblack) [inline src/pages/components/UI-shell-header/code.mdx:72]:
```suggestion
      style={{
```

File: src/pages/components/UI-shell-header/code.mdx [hunk 2]
```diff
@@ -84,7 +90,13 @@ the Storybooks for each framework below.
       src="https://codesandbox.io/embed/header-base-v07hj?fontsize=14&hidenavigation=1&view=preview"
       title="header-with-nav"
       allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media"
-      style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+      sstyle={{
```
Comment (joshblack) [inline src/pages/components/UI-shell-header/code.mdx:93]:
```suggestion
      style={{
```

File: src/pages/components/UI-shell-header/code.mdx [hunk 3]
```diff
@@ -99,7 +111,13 @@ the Storybooks for each framework below.
       src="https://codesandbox.io/embed/header-with-actions-vh014?fontsize=14&hidenavigation=1&view=preview"
       title="header-with-actions"
       allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media"
-      style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+      sstyle={{
```
Comment (joshblack) [inline src/pages/components/UI-shell-header/code.mdx:114]:
```suggestion
      style={{
```

File: src/pages/components/UI-shell-header/code.mdx [hunk 4]
```diff
@@ -114,7 +132,13 @@ the Storybooks for each framework below.
       src="https://codesandbox.io/embed/header-with-actions-and-nav-m0lo2?fontsize=14&hidenavigation=1&view=preview"
       title="header-with-actions-and-nav"
       allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media"
-      style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+      sstyle={{
```
Comment (joshblack) [inline src/pages/components/UI-shell-header/code.mdx:135]:
```suggestion
      style={{
```

**Review by joshblack — CHANGES_REQUESTED** [review]
Looking great! Just noticed some of the replacements seem to have `sstyle` instead of `style` and was curious if that was intentional. When looking at things like the UI Shell demo they seem to be acting odd so just wanted to bring it up.

**Comment by alisonjoseph** [comment]
oops 🤦‍♀️ Fixed @joshblack

**Review by joshblack — APPROVED** [review]

**Review by aledavila — APPROVED** [review]

**Pull request merged**